packages/netsuite-adapter/src/filters/element_references.ts

```diff
@@ -108,6 +116,51 @@ const generateServiceIdToElemID = async (
   .map(elem => getElementServiceIdRecords(elem))
   .reduce<ServiceIdRecords>((acc, records) => Object.assign(acc, records), {})
 
+const isFilePath = (path: string): boolean =>
+  osPath.extname(osPath.basename(path)) !== ''
+
+const resolveRelativePath = (absolutePath: string | undefined, relativePath: string): string =>
+  (isDefined(absolutePath) ? osPath.resolve(osPath.dirname(absolutePath), relativePath) : '')
+
+const updateDependenciesToAdd = (
+  foundReferences: string[],
+  fetchedElementsServiceIdToElemID: ServiceIdRecords,
+  elementsSourceServiceIdToElemID: ServiceIdRecords,
+  element: InstanceElement,
+): ElemID[] =>
+  foundReferences
+    .map(ref => (isFilePath(ref) ? resolveRelativePath(element.value?.path, ref) : ref))
```

Comment:
```suggestion
    .map(ref => (isFilePath(ref) ? resolveRelativePath(element.value.path, ref) : ref))
```
also, what about folders or files without extension?